graph-db/package.json

```diff
@@ -24,7 +24,7 @@
     "gremlin": "^3.4.10",
     "lodash": "^4.17.21",
     "node-fetch": "^2.6.1",
-    "serverless": "^2.38.0",
+    "serverless": "^2.46.0",
```

Comment:
Why don't we update to the latest serverless version at `2.59.0`?

Reply:
That will have fixes for all the vulnerabilities.

Reply:
2.59.0 introduces a different issue